pyproject.toml

```diff
@@ -50,6 +50,7 @@ dependencies = [
     "PyYAML==6.0",
     "pydantic-yaml==1.0",
     "psutil>=5.9.5,<6.0.0",
+    "noisepy-seis-io",
```

Comment:
I recommend adding a minimum version (e.g. `>=0.1.0`) once the noisepy-seis-io package is published